tests/integration/codesnippetfromselectedcells.ts

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
Since this is testing the code snippet extension, I suggest the new tests are located in `tests/integration/codesnippet.ts` instead of a separate test file.

Reply:
`tests/integration/codesnippet.ts` contains tests for the dialog that allows the user to create and edit snippets. 

These new tests are primarily for the notebook editor. I felt that they were different enough to have a separate file.

Reply:
That's a good point, although we have been writing integration tests on a single test file per extension, I'm thinking we would need to start breaking them apart in the future (ie. pipeline editor test file is getting huge). We could start a code-snippet directory inside `tests/integration/` to organize them better at least. 
@marthacryan @ajbozarth what do you think?

Reply:
I would agree we should split up our tests into different files and sort them into directories. I'm ok leaving this as is in this PR an creating a followup issue for next sprint to split up and restructure our current tests